site/_docs/reference.md

```diff
@@ -2732,6 +2732,7 @@ In the following:
 | p q | CONCAT(string [, string ]*)                  | Concatenates one or more strings, null is treated as empty string
 | m p | CONCAT_WS(separator, str1 [, string ]*)      | Concatenates one or more strings, returns null only when separator is null, otherwise treats null arguments as empty strings
 | q | CONCAT_WS(separator, str1, str2 [, string ]*)  | Concatenates two or more strings, requires at least 3 arguments (up to 254), treats null arguments as empty strings
+| s | CONCAT_WS(separator [, string | array(string)]+)  | Concatenates one or more strings or arrays, returns null only when separator is null, treats other null arguments as empty strings
```

Comment:
Please describe the difference about the CONCAT_WS(separator, str1 [, string ]*). Others good to me.

Reply:
Added more description in `reference.md`.

The argument of CONCAT_WS(separator, str1 [, string ]*) do not support array type, but Spark CONCAT_WS can support, and supports the mixed use of string and array.  
For example the test in `spark.iq`: `SELECT CONCAT_WS(',', 'a1', 'b1', 'c1', array('a'), array('b'), array('c'))`